iroh-relay/src/server/resolver.rs

```diff
@@ -0,0 +1,93 @@
+use std::{sync::Arc, time::Duration};
+
+use anyhow::{anyhow, Result};
+use reloadable_state::Reloadable;
+use rustls::{
+    server::{ClientHello, ResolvesServerCert},
+    sign::CertifiedKey,
+};
+use tokio_util::{sync::CancellationToken, task::AbortOnDropHandle};
+
+/// The default certificate reload interval.
+pub const DEFAULT_CERT_RELOAD_INTERVAL: u64 = 60 * 60 * 24;
```

Comment:
No units!  Maybe make it a `Duration`?  Or is that still not `const`?  Should definitely document the units though.